hardhat.config.ts

```diff
@@ -1,13 +1,14 @@
 import '@nomiclabs/hardhat-solpp';
-import 'hardhat-typechain';
 import '@nomiclabs/hardhat-ethers';
 import '@matterlabs/hardhat-zksync-solc';
+import '@typechain/hardhat'
+
 
 const systemConfig = require('./SystemConfig.json');
 
 export default {
     zksolc: {
-        version: '1.3.11',
+        version: '1.3.14',
```

Comment:
It is a change to system contracts (not only scripts)

Reply:
good point ! - changing the PR to point at DEV.